modules/react/checkbox/lib/CheckboxInput.tsx

```diff
@@ -65,29 +65,29 @@ const checkboxInputStencil = createStencil({
     marginTop: calc.negate(px2rem(3)),
     marginInlineStart: calc.negate(px2rem(3)),
     position: 'absolute',
-    opacity: 0,
+    opacity: system.opacity.zero,
 
     '&:not(:disabled)': {
       cursor: 'pointer',
     },
 
-    '&:where(:hover, .hover) ~ span:first-of-type': {
-      boxShadow: `0 0 0 ${px2rem(7)} ${base.soap200}`,
+    '&:where(:hover,.hover) ~ span:first-of-type': {
+      boxShadow: `0 0 0 ${px2rem(7)} ${system.color.bg.alt.soft}`,
```

Comment:
What token should we use for this? Not sure if `bg` should be used since it's on `boxShadow`

Reply:
this was discussed yesterday, and `bg` was suggested by design team